src/ItemManager.js

```diff
@@ -0,0 +1,52 @@
+
+class ItemManager {
+    constructor() {
+        this.tasks = [];
+        this.pokemonClient = new PokemonClient();
+    }
+
+    async handleTask(task) {
+        let taskSplit = task.split(",");
```

Comment:
In terms of responsibilities this logic should be on your main.js, the item manager should not be familiar with the UI, the structure of the input or anything, I would want that if you will drastically change the UI item manager would still be relevant and won't be affected